src/core_plugins/elasticsearch/lib/create_kibana_index.js

```diff
@@ -1,25 +0,0 @@
-export default function (server) {
```

Comment:
__question:__ I couldn't find any comments about this "feature", but just in general why did we need this previously and why we don't need this anymore? Mostly for my understanding.

Reply:
What I call the "saved objects index" used to be the "Kibana index", which needed to be created before the first write or elasticsearch would automatically create the index with automatic (and likely incorrect) mappings.

This part of the health checker was responsible for doing that, and we would run it on an interval, but this new strategy teaches elasticsearch how to automatically create the index correctly instead.

Reply:
Got it, thanks